flask_sqlalchemy/__init__.py

```diff
@@ -403,6 +403,11 @@ def iter_pages(self, left_edge=2, left_current=2,
                 yield num
                 last = num
 
+    def as_dict(self):
+        """Returns dict representation of the object"""
```

Comment:
This needs a better doc string. Something like "Create a dict with the pagination fields for use as a JSON response."